Sources/StreamChat/Workers/CurrentUserUpdater.swift

```diff
@@ -161,6 +161,33 @@ class CurrentUserUpdater: Worker {
             completion?($0.error)
         }
     }
+    
+    /// Get all blocked users.
+    ///
+    /// - Parameter completion: Called when the API call is finished. Called with `Error` if the remote update fails.
+    ///
+    func getBlockedUsers(completion: @escaping (Result<[BlockedUser], Error>) -> Void) {
+        apiClient.request(endpoint: .loadBlockedUsers()) {
+            switch $0 {
+            case let .success(payload):
+                var blockedUsers: Set<BlockedUserDTO> = []
```

Comment:
This needs to be model, not DTO
```suggestion
                var blockedUsers: Set<BlockedUser> = []
```

Reply:
I was able to bypass the compiler errors only by making it an Array 🤔
```swift
var blockedUsers: [BlockedUser] = []
```